roofit/roofit/src/RooLandau.cxx

```diff
@@ -72,7 +73,24 @@ void RooLandau::computeBatch(cudaStream_t* stream, double* output, size_t nEvent
 Int_t RooLandau::getGenerator(const RooArgSet& directVars, RooArgSet &generateVars, Bool_t /*staticInitOK*/) const
 {
   if (matchArgs(directVars,generateVars,x)) return 1 ;
-  return 0 ;
+  return 0;
+}
+
+Int_t RooLandau::getAnalyticalIntegral(RooArgSet &allVars, RooArgSet &analVars, const char * /*rangeName*/) const
+{
+   if (matchArgs(allVars, analVars, x))
+      return 1;
+   return 0;
+}
+
+Double_t RooLandau::analyticalIntegral(Int_t code, const char *rangeName) const
+{
+   assert(code == 1);
+
+   double max = x.max(rangeName);
+   double min = x.min(rangeName);
+
+   return ROOT::Math::landau_cdf(max, sigma, mean) - ROOT::Math::landau_cdf(min, sigma, mean);
```

Comment:
You have to multiply by sigma the result of `RooLandau::analyticalIntegral` as `RooLandau::evaluate` uses `TMath::Landau()` with `norm=kFalse`

Reply:
Thanks for spotting this! You are right of course